modules/coach/src/main/CoachPager.scala

```diff
@@ -27,7 +27,7 @@ final class CoachPager(
         def nbResults: Fu[Int] = fuccess(9999)
 
         def slice(offset: Int, length: Int): Fu[List[Coach.WithUser]] =
-          coachRepo.coll
+          userRepo.coll
```

Comment:
no, you want to run the aggregation on the coach collection, and lookup the user collection.